website/blog/2025-04-20-vs-code-with-podman-desktop.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Supercharge Your Container Development in VS Code with Podman and Podman Desktop
+description: How to develop apps with VS Code and Podman Desktop
+authors: [firewall]
+tags: [podman-desktop, podman, vscode, docker, extension, development, container, tools]
+hide_table_of_contents: false
+---
+
+Developing containerized applications can sometimes feel complex, but with the right tools, it can be a smooth and efficient process. In this blog post, we'll explore how to leverage the power of Visual Studio Code (VS Code) together with Podman and Podman Desktop to streamline your container development workflow. We'll cover setting up the necessary tools and using the VS Code extensions for a seamless experience.
```

Comment:
>We'll cover setting up the necessary tools and using the VS Code extensions for a seamless experience. 

This is a bit misleading as it's unlikely that a developer will need the two extensions. The blog post provides an overview of the existing extensions, to provide a seamless experience between VSCode and Podman.